www/www-deploy.yml

```diff
@@ -2,37 +2,26 @@
 
 - hosts: www
 
-  pre_tasks:
-
-    - name: NGINX - SSL File Deployment
-      become: yes
-      copy:
-        dest="{{ item.key }}"
-        content="{{ item.value.content }}"
-        owner="{{ item.value.owner }}"
-        group="{{ item.value.group }}"
-        mode="{{ item.value.mode }}"
-      with_dict: "{{ nginx_ssl_cert_files }}"
-      no_log: true
-
   roles:
-
+    - role: openmicroscopy.ssl-certificate
     - role: openmicroscopy.nginx-proxy
       tags: nginxconf
-      nginx_proxy_worker_processes: "{{ ((ansible_processor_count * ansible_processor_cores) / 2) |round|int }}"
-      nginx_proxy_worker_connections: 65000
-      nginx_proxy_ssl: True
-      nginx_proxy_ssl_certificate: "{{ nginx_ssl_files_path }}/{{ nginx_ssl_cert_filename }}"
-      nginx_proxy_ssl_certificate_key: "{{ nginx_ssl_files_path }}/{{ nginx_ssl_key_filename }}"
-      nginx_proxy_http2: True
-      nginx_proxy_force_ssl: False
-      nginx_proxy_404: "/404.html"
-      nginx_proxy_conf_http:
-        - "client_max_body_size 2g"
 
   vars:
-
+    ssl_certificate_public_path: /etc/nginx/ssl/server.crt
```

Comment:
These could have also been left as the default, no?

Reply:
I used the application-specific SSL directory to minimize confusion after discussion with @kennethgillen. Using the default should equally work.
Since the variable is coupled to `nginx_proxy_ssl_certificate` below, my impression is that explicitly declaring its value prevents issues if the defaults were to change.

Reply:
Ah, I just found the discussion around https://serverfault.com/a/840191 -- if this is the reasoning, should be not adjust the upstream role to something like `/etc/ssl/local_certs`?

Reply:
@joshmoore - `/etc/ssl/local_certs` could work, to be application-agnostic.

Reply:
See https://github.com/openmicroscopy/ansible-role-ssl-certificate/pull/5 for the proposed role change.